### DISCUSSION TITLE
Bagaimana cara Block IP menggunakan AWS EC2?

### DISCUSSION THREAD

**Discussion by dapealfy** [awsugid/forum#32] [Q&A]
Halo guys, izin nanya ada yang pernah block ip address ga pake aws ec2? soalnya ini ada yang access server saya tapi ngespam banget

**Comment by xepatan** [comment]
cobain ulik-ulik di security groupnya, karena firewallny ada disana...

**Comment by za** [comment]
Halo @dapealfy 

Salah satu cara membatasi akses ke EC2 bisa menggunakan security group. 

> Soalnya ini ada yang access server saya tapi ngespam banget

Ini akses ke server di port 80 kah? Apakah IP address spammer statis?
